src/main/java/net/coderbot/iris/uniforms/Uniforms.java

```diff
@@ -37,6 +39,12 @@ public Uniforms(GlProgram program) {
 		gbufferProjection = GL21.glGetUniformLocation(programId, "gbufferProjection");
 		gbufferProjectionInverse = GL21.glGetUniformLocation(programId, "gbufferProjectionInverse");
 
+		viewHeight  = MinecraftClient.getInstance().getWindow().getHeight();
```

Comment:
This isn't quite right. `private int viewHeight` as declared above actually holds the location of the viewHeight uniform used for sending values to the shaders. I have a few ideas on how to refactor the uniform code to be a bit more intuitive, since right now it's a very basic implementation.

This specific call should probably be happening in a `glUniform*` call within `update()`.